docs/tutorials/how-tos/use-batch-transactions/send-a-multi-account-batch-transaction.md

```diff
@@ -0,0 +1,172 @@
+---
+seo:
+    description: Send a Batch transaction containing transactions from multiple accounts. 
+metadata:
+  indexPage: true
+labels:
+  - Batch
+  - Transactions
+---
+# Send a Multi-Account Batch Transaction
+
+This tutorial shows you how to create a [Batch transaction][] containing transactions from multiple accounts, where each account must sign the Batch transaction. Any account, even one not involved in the inner transactions, can submit the batch.
+
+## Goals
+
+By the end of this tutorial, you will be able to:
+
+- Create a Batch transaction with multiple inner transactions, signed by multiple accounts, and submitted by a third party account.
+- Configure the Batch transaction to ensure atomicity, so that either all inner transactions succeed or they all fail.
```

Comment:
```suggestion
- Create a `Batch` transaction with multiple inner transactions, signed by multiple accounts, and submitted by a third-party account.
- Configure the `Batch` transaction to ensure atomicity, so that either all inner transactions succeed or they all fail.
```